client/src/components/footer/footer.js

```diff
@@ -185,17 +185,19 @@ class Footer extends Component {
             style={{ display: display ? "block" : "none" }}
           >
             <button
-              className="btn tw-bg-labLightGray btn-xl text-uppercase back "
+              //className="btn tw-bg-labLightGray btn-xl text-uppercase back "
+              className="back btn tw-w-32 tw-font-medium tw-bg-bgwhite text-uppercase  tw-border-b-labYellow tw-border-l-labYellow tw-border-8 tw-shadow-none"
               onClick={() => handleRedirect(actions, lab, body - 1)}
               style={{
                 display:
                   this.disappearBack(body) || hideOnLanding ? "none" : "block",
               }}
             >
-              Previous -{" "}
+              Back
```

Comment:
Make sure you pull in the `development/ui-refresh` branch into this branch so you can use the tailwind config file with the new style guidelines

Reply:
Edit: this is wrong, no need to worry about this!